docs/pages/push-notifications/push-notifications-setup.mdx

```diff
@@ -1,28 +1,79 @@
 ---
-title: Push Notifications Setup
+title: Push notifications setup
 sidebar_title: Setup
 ---
 
 import { Tab, Tabs } from '~/ui/components/Tabs';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { BoxLink } from '~/ui/components/BoxLink';
 
-To get the client-side ready for push notifications, the 2 main things we need are:
+To use Expo's push notification service, you need to configure your app to install a set of libraries, add functions to handle notifications and configure credentials for Android and iOS. After going through these steps, you'll be able to test notification sending and receiving notifications on a device.
+
+To get the client-side ready for push notifications, the following is required:
 
 - The user's permission to send them push notifications
-- The user's ExpoPushToken- if push notifications are mail, then the ExpoPushToken is the user's address.
+- The user's [`ExpoPushToken`](/versions/latest/sdk/notifications/#expopushtoken) &mdash; if push notifications are mail, then the `ExpoPushToken` is the user's address.
+
+You'll need to use [`expo-notifications`](/versions/latest/sdk/notifications) library to request for user's permission and fetch the `ExpoPushToken`. You'll also use [`expo-device`](/versions/latest/sdk/device) to check whether the app is running on a physical device. The `expo-notifications` library is not supported on an Android Emulator or an iOS simulator.
```

Comment:
```suggestion
You'll need to use the [`expo-notifications`](/versions/latest/sdk/notifications) library to request a user's permission and to fetch the `ExpoPushToken`. You'll also use [`expo-device`](/versions/latest/sdk/device) to check whether the app is running on a physical device. The `expo-notifications` library is not supported on an Android Emulator or an iOS simulator.
```
I think that this information would be much better next to the step where they are installed below. So instead of summarizing how something works here, then having a step below that installs it, we should (1) ask the developer to install the libraries and (2) explain that they won't work on simulators/emulators.

Reply:
Initially, I had the explanation after installation. I don't exactly remember why I changed it 😄 
I've now added the explanation back in the installation step.